lib/config/configItems.joi.js

```diff
@@ -132,6 +132,13 @@ const mongoJoi = joi.object({
     }),
 });
 
+const qpKafkaJoi = joi.object({
+    hosts: joi.string().required(),
+    topic: joi.string().required(),
+    groupId: joi.string().required(),
```

Comment:
should these be `required`?
it may seem appropriate to have some defaults here, instead of just in zkop (and could help this code gets partially integrated, e.g. not all parts are done)